rotkehlchen/assets/exchanges_mappings/kraken.py

```diff
@@ -223,4 +223,8 @@
     evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'TBTC',  # noqa: E501
     evm_address_to_identifier('0x5283D291DBCF85356A21bA090E6db59121208b44', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'BLUR',  # noqa: E501
     'HDX': 'HDX',
+    evm_address_to_identifier('0xC581b735A1688071A1746c968e0798D642EDE491', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'EURT',  # noqa: E501
+    evm_address_to_identifier('0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a', ChainID.ARBITRUM, EvmTokenKind.ERC20): 'GMX',  # noqa: E501
+    'SUI': 'SUI',
+    # evm_address_to_identifier('0xcf0C122c6b73ff809C693DB761e7BaeBe62b6a2E', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'FLOKI',  # noqa: E501
```

Comment:
Why is this commented out?

```suggestion
    # evm_address_to_identifier('0xcf0C122c6b73ff809C693DB761e7BaeBe62b6a2E', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'FLOKI',  # noqa: E501
```

Reply:
is not needed. Removed it